package.json

```diff
@@ -102,8 +105,10 @@
     "build": "DISABLE_ESLINT_PLUGIN=true BROWSER=none GENERATE_SOURCEMAP=true node scripts/build.js",
     "build-windows": "set \"DISABLE_ESLINT_PLUGIN=true\" && set \"BROWSER=none\" && set \"GENERATE_SOURCEMAP=true\"  && node scripts/build.js",
     "test": "node scripts/test.js",
+    "test-mocha": "NODE_ENV=test ./node_modules/.bin/mocha --reporter spec --recursive",
     "download-value-sets": "node src/services/valuesets/updateValueSetDb.mjs $UMLS_API_KEY",
-    "serve": "serve ./build --ssl-cert https.crt --ssl-key https.key"
+    "serve": "serve ./build --ssl-cert https.crt --ssl-key https.key",
+    "convert-csv": "node src/hooks/useCds/convert-strides.mjs"
```

Comment:
It may be nice to add a very brief README description describing how to run `convert-csv`, and what it does.